### PR TITLE
[fix] prepare_dict support english and chinese in one lexicon.txt

### PR DESCRIPTION
**Pull request by CuiDonCai**
tools/fst/prepare_dict.py, it can't split chinese phrase into character when bpemodel given [issus](https://github.com/wenet-e2e/wenet/issues/1653).  Now It can process english and chinese in one lexicon.txt .

### CODE REVIEW

File: tools/fst/prepare_dict.py [hunk 1]
```diff
@@ -41,7 +41,10 @@ def contain_oov(units):
             if word in lexicon_table:
                 continue
             if bpemode:
-                pieces = sp.EncodeAsPieces(word)
+                if word.encode('utf8').isalpha():
+                    pieces = sp.EncodeAsPieces(word)
+                else:
+                    pieces = word
                 if contain_oov(pieces):
```
[the remark this answers is not in the document]
Comment (CuiDonCai) [reply tools/fst/prepare_dict.py:48]: sure.

[the remark this answers is not in the document]
Comment (CuiDonCai) [reply tools/fst/prepare_dict.py:48]: em, where do you mean add comments?

[the remark this answers is not in the document]
Comment (xingchensong) [reply tools/fst/prepare_dict.py:48]:
```py
# We assume that the lexicon does not contain code-switch, i.e., the word contains both English and Chinese.
# see PR https://github.com/wenet-e2e/wenet/pull/1693
# and Issue https://github.com/wenet-e2e/wenet/issues/1653
if word.encode('utf8').isalpha():
    pieces = sp.EncodeAsPieces(word)
else:
    pieces = word
```

**Comment by robin1001** [comment]
please fix the lint problem.

**Comment by robin1001** [comment]
./tools/fst/prepare_dict.py:44:125: B950 line too long (124 > 80 characters)

**Review by robin1001 — APPROVED** [review]

**Pull request merged**